website/docs/r/application_gateway.html.markdown

```diff
@@ -369,6 +372,22 @@ A `custom_error_configuration` block supports the following:
 
 * `custom_error_page_url` - (Required) Error page URL of the application gateway customer error.
 
+---
+
+A `redirect_configuration` block supports the following:
+
+* `name` - (Required) Unique name of the redirect configuration block
+
+* `redirect_type` - (Required) The type of redirect. Possible values are `Permanent`, `Temporary`, `Found` and `SeeOther`
+
+* `target_listener_name` - (Optional) The name of the listener to redirect to. This parameter is mutually exclusive with `target_url`.
+
+* `target_url` - (Optional) The Url to redirect the request to. This parameter is mutually exclusive with `target_listener_name`.
```

Comment:
for consistency with other resources, can we make this:

```suggestion
* `target_url` - (Optional) The Url to redirect the request to. This field must be set when `target_listener_name` is set.
```

Reply:
Done